### PR TITLE
Adds crusader armour pack to cargo.

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: Templar GuyonBroadway
add: Just in case the Saracens come back cargo can now order a crate stuffed with crusader armour. Comes in every colour so you can live out your first crusade/power ranger crossover fantasy.
/:cl:

We used to have this in old hippie and it wuz great. 
https://youtu.be/7U9lVjZg-Kg

### CODE REVIEW

**Comment by ghost** [comment]
yeah i love it +1

**Comment by earthmoonjune** [comment]
+1

**Comment by SplergeBucket** [comment]
sure
tho unless there's some way that this can require the chaplain
i'd prefer if the armour was weaker than the one the chaplain can get
am i remembe this right????

**Comment by ghost** [comment]
nah it was p. much the same as chaplain but chaplain's sprites are unique

**Comment by SplergeBucket** [comment]
ye thought so
i dont like the removal of something unique for the chaplainos
so if its a weaker version for the followers im down

**Comment by KittyKathy** [comment]
Nanotrasen would like to remind you that we pay you to work, not prance around wearing crusader armour and being a [LARPer](https://en.wikipedia.org/wiki/Live_action_role-playing_game). We have removed this silly little dress-up game from your cargo packs. GET BACK TO WORK!

> Comes in every colour so you can live out your first crusade/power ranger crossover fantasy.

Removing anime cancer #5288 and replacing it with LARPer cancer - GuyonBroadway 2017

Why would he do this? I'll tell you why: so he can LARP as a cargo tech and power creep cargo.

[`/obj/item/clothing/suit/armor/riot/knight`](https://github.com/HippieStation/HippieStation/blob/master/code/modules/clothing/suits/armor.dm#L400)
The Chaplin's Knight armor and these colored variations are exactly the same as riot armor except for the  name, icons and description.



`/obj/item/clothing/head/helmet/riot`
[`armor = list(melee = 45, bullet = 15, laser = 5,energy = 5, bomb = 5, bio = 2, rad = 0, fire = 50, acid = 50)`](https://github.com/HippieStation/HippieStation/blob/master/code/modules/clothing/head/helmet.dm#L65)
`/obj/item/clothing/head/helmet/knight`
[`armor = list(melee = 41, bullet = 15, laser = 5,energy = 5, bomb = 5, bio = 2, rad = 0, fire = 0, acid = 50)`](https://github.com/HippieStation/HippieStation/blob/master/code/modules/clothing/head/helmet.dm#L201)
The Helmets are different than riot helmets but armor wise pretty close.


[`/obj/item/shield/riot/roman`](https://github.com/HippieStation/HippieStation/blob/master/code/game/objects/items/shields.dm#L40)
Same case for the shields as with the armor, exactly the same as riot shields except for the name, icons and description.



That was just the armor now onto the cargo pack itself.

Crusader Armour Pack
Cost: 2500 for 4 sets(armor,helmet,shield) of basically the equivalent of riot armor all in a crate requiring no special access to open.

Riot Gear Packs-riot armor is split up into 3 separate crates for each piece, each requiring security access.

Riot Helmets Crate
[`/datum/supply_pack/security/armory/riothelmets`](https://github.com/HippieStation/HippieStation/blob/master/code/modules/cargo/packs.dm#L296)
Cost: 1500 for 3 riot helmets only.

Riot Armor Crate
[`/datum/supply_pack/security/armory/riotarmor`](https://github.com/HippieStation/HippieStation/blob/master/code/modules/cargo/packs.dm#L304)
Cost: 1500 for 3 riot armor only.

Riot Shields Crate
[`/datum/supply_pack/security/armory/riotshields`](https://github.com/HippieStation/HippieStation/blob/master/code/modules/cargo/packs.dm#L312)
Cost: 2000 for 3 riot shields only.

Total cost: 5000 for 3 sets of actual riot gear(riot shotgun not included, weird you can't order that), shipped in 3 crates each requiring security access or a means to break the lock.

In conclusion, this pack is basically half off the total cost for 3 riot gear sets with no access lock and a bonus gear set(4 sets in total instead of 3 sets).
**P O W E R C R E E P**.

**Comment by GuyonBroadway** [comment]
After this imma make a balance pass on cargo crate  costs because they are all overpriced.

**Comment by JohnGinnane** [comment]
@GuyonBroadway @KittyKathy raised some good points. As hilarious as it would be for sec to order crusader armour to fight off revs or aliums, why would they ever order (more) riot gear? 

Maybe these crates should require Chaplain, HoP, or Captain access to open. Maybe increase the price of the crusader crate and/or reduce the price of the riot gear?

**Review by JohnGinnane — DISMISSED** [review]
✝️ ⚔️

**Comment by GuyonBroadway** [comment]
Imma simply lower the riot gear costs in another pr. Nobody ever buys that anyway.

**Comment by Pyko1** [comment]
Why not in this to make it all balanced out tho?
As it is this pr is currently pretty unbalanced.

And imho feature balance issues should not be swept under a rug with ill just fix it in future PR.

**Comment by GuyonBroadway** [comment]
I wont make it in this PR because atomic PR's.

**Comment by ghost** [comment]
love it

**Comment by GuyonBroadway** [comment]
Imma rebalance this and the riot gear by make each the shields, armour and helmets cost about 250 each. 
So this crate will now cost 3000 (four armour = 1000 four shields = 1000 four helmets = 1000) and will get another PR up to rebalance the other armour crates.

I might just make a lot of cargo crap cheaper in general because cargo get no passive points these days.

**Review by ghost — APPROVED** [review]
